packages/Provider/src/intl/en-US.json

```diff
@@ -0,0 +1,3 @@
+{
+    "testMessage": "test message"
```

Comment:
I wouldn't add test messages in the intl folder. We might end up with a mix and match of real and tests values, and they are all going to be sent to the users. I suggest you create a local object in your test file instead